README.md

```diff
@@ -30,7 +34,7 @@ Tapetti is a wallpaper manager that you never asked for. Feature-rich yet minima
 
 </div>
 
-# Featuers
+## Features
```

Comment:
Why did you add a extra `#` though? 😅

Reply:
It's usually good practice to remain the `#`/`<h1>` for titles for search engines and stuff, I don't think it matters for github but it's still better practice to use more specific headings.

Reply:
I see.